tools/cldr-apps/src/main/java/org/unicode/cldr/web/Dashboard.java

```diff
@@ -64,33 +52,42 @@ public ReviewNotification add(ReviewNotification notification) {
             return notification;
         }
 
+        @Schema(description = "Notifications that the user has chosen to hide")
+        private HiddenNotifications hidden;
+
         /**
-         * Notifications that the user has chosen to hide
+         * Get a map from each subtype (String from CheckCLDR.CheckStatus.Subtype)
+         * to an array of path-value pairs
+         *
+         * @return the map
          */
-        public HashMap<String, List<String>> hidden;
+        public Map<String, PathValuePair[]> getHidden() {
+            return hidden.getHidden();
```

Comment:
For single-path requests, getHidden and voterProgress are superfluous -- for performance, I intend to bypass them in a subsequent PR